model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/JpaUpdate25_0_0_MySQL_ConsentConstraints.java

```diff
@@ -35,7 +35,7 @@ protected void generateStatementsImpl() throws CustomChangeException {
                         " AND uc.USER_ID = max_dates.USER_ID AND uc.LAST_UPDATED_DATE = max_dates.MAX_UPDATED_DATE)"
         ));
         statements.add(new RawSqlStatement(
-                "  CREATE TEMPORARY TABLE TEMP_USER_CONSENT_IDS" +
+                "  CREATE TABLE TEMP_USER_CONSENT_IDS" +
```

Comment:
The corresponding statement to drop the table is still `DROP TEMPORARY TABLE IF EXISTS TEMP_USER_CONSENT_IDS`. Shouldn't it be `DROP TABLE IF EXISTS TEMP_USER_CONSENT_IDS`?

Similar to what you did in the following change.

Reply:
Yeah I missed one :D